### PR TITLE
Bl orient 827

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient827.xml [hunk 1]
```diff
@@ -423,76 +479,76 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                                 <layout columns="2" writtenLines="23 25" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is neatly written of the <date>18th century.</date>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient827.xml:532]: Smth is wrong in this sentense?

File: LondonBritishLibrary/orient/BLorient827.xml [hunk 2]
```diff
@@ -423,76 +479,76 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                            </q>
                                 </item>
-                           
-                                  
+
+
                                        <item xml:id="e1">
                                            <locus from="1r" to="3r"/>
                                            <desc>trials of the pens and a few notes in Amharic.</desc>
                                            <q xml:lang="gez">ባርካ፡ እግዚኦ፡ ለዛቲ፡ ብርዕ፡
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient827.xml:555]: Trials

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient827.xml (hunk 2 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient827.xml:555]: Sorry, I do not understand. I should delete "Pen" at the beginning?

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient827.xml [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 schematypens="http://relaxng.org/ns/structure/1.0"
-?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
    schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BLorient827" type="mss">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Life of Alexander the Great, the story of Secundus, Discourse of our Lord, History of Bejamin and a chronographic work</title>
+                <title>Zenā ʾƎskǝndǝr, zenā Sǝkǝndǝs, zenā Sarkis,
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient827.xml:8]: I would spell each zenā with a capital and bəḍuʿ without shwa at the end

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

**Review by thea-m — APPROVED** [review]

**Pull request merged**